src/round.rs

```diff
@@ -85,14 +88,105 @@ fn span_for_digits(digits: u16) -> u32 {
     }
 }
 
+/// Extension trait for rounding or truncating a DateTime by a Duration.
+///
+/// # Limitations
+/// Both rounding and truncating are done via [`Duration::num_nanoseconds`] and
+/// [`DateTime::timestamp_nanos`]. This means that they will fail if either the
+/// `Duration` or the `DateTime` are too big to represented as nanoseconds. They
+/// will also fail if the `Duration` is bigger than the timestamp.
+///
+/// [`Duration::num_nanoseconds`]: struct.Duration.html#method.num_nanoseconds
+/// [`DateTime::timestamp_nanos`]: struct.DateTime.html#method.timestamp_nanos
```

Comment:
This shouldn't be necessary: docs are built with nightly and should get auto-linkified if the item is in scope. You can check by running `cargo +nightly doc --open` -- it should run without warnings, and hte link should work without this.

It's also better to use the auto-linkification, because it is robust to the fact that items can appear in multiple places in docs, and relative paths aren't always correct.

Reply:
Ace, I've removed them.